meson.build

```diff
@@ -10,7 +10,8 @@ project('f90wrap',
 import('fs').read('f90wrap/__init__.py')
 
 # Adding at project level causes many spurious -lgfortran flags.
-add_languages('fortran', native: false)
+# Make Fortran optional since f90wrap itself doesn't need it (only projects using f90wrap do)
```

Comment:
I'm not sure I understand, doesn't f90wrap need fortran to compile sizeof_fortran_t when building itself?